stacks/lemp/service-config/mime.types

```diff
@@ -0,0 +1,89 @@
+
```

Comment:
I don't love having `mime.types` in `vagrant-spk`, but I guess it makes more sense than copying it in from the Debian VM.